depmap-shared/color_palette.json

```diff
@@ -4,13 +4,13 @@
   "rnai_color": "#52288E",
   "expression_color": "#7B8CB2",
   "copy_number_color": "#CEA3CB",
-  "gdsc_color": "#006600",
-  "ctrp_color": "#99cc66",
-  "repurp_color": "#669933",
-  "rep1m_color": "#547335",
-  "rep_all_single_pt_color": "#55aa24",
+  "gdsc_color": "#c55252",
+  "ctrp_color": "#c55252",
+  "repurp_color": "#c55252",
+  "rep1m_color": "#c55252",
+  "rep_all_single_pt_color": "#c55252",
```

Comment:
A question for reviewers: Previously, we used a slightly different shade of green for each dataset. Do we still care about using different shades of the compound color per dataset, or is it fine to leave these all as the same red color?